src/Native/MatrixFactorizationNative/UnmanagedMemory.cpp

```diff
@@ -0,0 +1,36 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#include <malloc.h>
```

Comment:
Should we be using stdlib.h instead?

[https://www.quora.com/What-is-the-header-file-that-includes-malloc-and-calloc](https://www.quora.com/What-is-the-header-file-that-includes-malloc-and-calloc)
[https://stackoverflow.com/questions/12973311/difference-between-stdlib-h-and-malloc-h](https://stackoverflow.com/questions/12973311/difference-between-stdlib-h-and-malloc-h)